### PR TITLE
Debug dhx

### PR DESCRIPTION
**Pull request by AdamIsrael**
Edits for papercuts, readability and clarity

### CODE REVIEW

File: src/en/developer-debug-dhx.md [hunk 1]
```diff
@@ -43,8 +40,8 @@ Here is an overview of the features:
 
 First, follow [the instructions in the README](https://github.com/juju/plugins#install)
 to install the plugin bundle.
-This should be straightforward and will give you access to DHX as well as all
-the other very useful Juju plugins in the bundle.
+This should be straightforward and will give you access to all of the
+juju plugins, DHX included.
```
Comment (chuckbutler) [inline src/en/developer-debug-dhx.md]: s/juju/Juju

**Comment by chuckbutler** [comment]
One minor nit, otherwise :+1: LGTM

**Comment by mbruzek** [comment]
+1 LGTM Thanks for cleaning this up Adam!

File: src/en/developer-debug-dhx.md [hunk 2]
```diff
@@ -1,13 +1,11 @@
...
-[DHX](https://github.com/juju/plugins/blob/master/juju-dhx) is a Juju plugin
-that allows you to fully and automatically customise the machines created by
-Juju, making developing and debugging hooks as painless as possible.
-
-It is a drop in replacement for
-[`juju debug-hooks`](./developer-debugging.html#the-'debug-hooks'-command).
-It is still a plugin and not yet integrated into Juju itself, so currently
-considered beta quality. DHX is recommended for more advanced users who need to
+[DHX](https://github.com/juju/plugins/blob/master/juju-dhx) is a plugin --
+scripts that extend the functionality of Juju -- that allows you to fully and
```
Comment (marcoceppi) [inline src/en/developer-debug-dhx.md]: `that allows you` seems odd to me, `which allows you` sounds more natural*

*IANAEM

**Pull request merged**